src/domain/relay/limit-addresses.mapper.ts

```diff
@@ -228,28 +228,15 @@ export class LimitAddressesMapper {
     chainId: string;
     address: `0x${string}`;
   }): boolean {
-    const multiSendCallOnlyDeployment = getMultiSendCallOnlyDeployment({
-      version: args.version,
-      network: args.chainId,
-    });
-
-    const isCallOnly =
-      multiSendCallOnlyDeployment?.networkAddresses[args.chainId] ===
-        args.address ||
-      multiSendCallOnlyDeployment?.defaultAddress === args.address;
+    const multiSendCallOnlyDeployments = getMultiSendCallOnlyDeployments(args);
+    const isCallOnly = multiSendCallOnlyDeployments.includes(args.address);
 
     if (isCallOnly) {
       return true;
     }
 
-    const multiSendCallDeployment = getMultiSendDeployment({
-      version: args.version,
-      network: args.chainId,
-    });
-    return (
-      multiSendCallDeployment?.networkAddresses[args.chainId] ===
-        args.address || multiSendCallDeployment?.defaultAddress === args.address
-    );
+    const multiSendCallDeployments = getMultiSendDeployments(args);
+    return multiSendCallDeployments.includes(args.address);
```

Comment:
Nit: wdyt about simplifying this a bit by:

```typescript
return (
      getMultiSendCallOnlyDeployments(args).includes(args.address) ||
      getMultiSendDeployments(args).includes(args.address)
    );
```

Reply:
Agreed. Changed in 79fa30cc6d681b6b4f8ceaed1d063464d5ade42e.